app/controllers/system_messages_controller.rb

```diff
@@ -2,6 +2,8 @@ class SystemMessagesController < ApplicationController
   before_action :authenticate_user!
   before_action :set_system_message, only: %i[show]
 
+  ensure_authorization_performed
```

Comment:
Was this needed for some reason? If so, we might need it other places.

Reply:
removed